### ISSUE TITLE
HomeConnect API unexpected values (WAV28KH3GB/01)

### ISSUE THREAD

**Issue by badgertastic** [thoukydides/homebridge-homeconnect#213]
### Plugin Version

0.37.8

### Home Connect Appliance(s)

Bosch Washer WAV28KH3GB/01

### Log File

```shell
==========================================================================================
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] 
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] // Union types
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] export type ProcessPhase =
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Common.EnumType.ProcessPhase.IntermediateSpin'; // (unrecognised)
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] 
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] // Option
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] export interface OptionValues {
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.Duration'?:                         number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.EnergyForecast'?:                   number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.EstimatedTotalProgramTime'?:        number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.FinishInRelative'?:                 number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.ProgramProgress'?:                  number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.RemainingProgramTime'?:             number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.RemainingProgramTimeIsEstimated'?:  boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'BSH.Common.Option.WaterForecast'?:                    number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Common.Option.LoadRecommendation'?:       number;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Common.Option.ProcessPhase'?:             ProcessPhase; // (unrecognised)
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.DryingTarget'?:              DryingTarget;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.DryingTargetAdjustment'?:    DryingTargetAdjustment;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.Gentle'?:                    boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.WrinkleGuard'?:              WrinkleGuard;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.IDos1.Active'?:             boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.IDos2.Active'?:             boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.Prewash'?:                  boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.RinseHold'?:                boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.RinsePlus1'?:               boolean;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.SpinSpeed'?:                SpinSpeed;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.Temperature'?:              Temperature;
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] }
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] 
[1/5/2024, 12:27:45 PM] [homebridge-homeconnect] ==========================================================================================
```

**Comment by thoukydides** [comment]
Thanks for the report.
That extra Washer ProcessPhase will be in the next release.

**Comment by thoukydides** [comment]
Should be fixed in [v0.37.9](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v0.37.9).
